main.js

```diff
@@ -1,25 +1,27 @@
-const words = []; //the list will contain the added words
+const words = []; 
 
-function addWords() {
-    let userInput = document.getElementById("userInput").value;
-    words.push(userInput);
+function addWords(formData) {
+    if (words.includes(formData.input.value)) {
```

Comment:
This does not seem right, we can still add more words of the same input.

![image](https://user-images.githubusercontent.com/75535042/213868651-7f191a04-62b1-4bed-956c-12ca002f6da0.png)

Reply:
This is the last version. The improved one does not have this problem. I think you've watched the wrong branch. :D